src/core/namespace.c

```diff
@@ -3888,6 +3960,13 @@ static const char* const proc_subset_table[_PROC_SUBSET_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(proc_subset, ProcSubset);
 
+static const char* const private_bpf_table[_PRIVATE_BPF_MAX] = {
+        [PRIVATE_BPF_NO]    = "no",
+        [PRIVATE_BPF_YES]   = "yes",
+};
+
+DEFINE_STRING_TABLE_LOOKUP(private_bpf, PrivateBPF);
```

Comment:
Please use `DEFINE_STRING_TABLE_LOOKUP_WITH_BOOLEAN()` to accept e.g. `true/false`, `1/0`.